examples/standalone/runfile/dynamics.py

```diff
@@ -182,7 +197,16 @@ def write_global_timings(experiment, filename, comm, root=0):
                 timer,
             )
 
+    if profiler is not None:
+        profiler.disable()
+
     timer.stop("total")
+
+    # output profiling data
+    comm.Barrier()
```

Comment:
Can we get by with one barrier? No comm in between I guess...

Reply:
Will remove.